manim/mobject/number_line.py

```diff
@@ -518,25 +560,24 @@ def add_labels(
         self.add(labels)
         return self
 
-    @staticmethod
-    def _create_label_tex(label_tex) -> "Mobject":
-        """Checks if the label is a ``float``, ``int`` or a ``str`` and creates a :class:`~.MathTex`/:class:`~.Tex` label accordingly.
+    def _create_label_tex(self, label_tex: Union[str, float, VMobject]) -> VMobject:
+        """Checks if the label is a :class:`~.VMobject`, otherwise, creates a label according to the label_constructor.
```

Comment:
Breaking the line and minor improvement to formatting.

```suggestion
        """Checks if the label is a :class:`~.VMobject`, otherwise, creates a 
        label according to the ``label_constructor``.
```